app/src/auth/utils/getSessionToken.js

```diff
@@ -2,9 +2,22 @@
 
 import StreamrClient from 'streamr-client'
 
-export default async (auth: Object): Promise<?string> => (
-    new StreamrClient({
-        restUrl: process.env.STREAMR_API_URL,
-        auth,
-    }).session.getSessionToken()
-)
+const parseError = async (error) => {
```

Comment:
If the entire body is wrapped in a try/catch, does it still need the `async` + `return await`?

Reply:
Yep. Doesn't look like it needs it. Guess I started w/ something different and that `async` just slipped in. Will revisit.